pkg/action/cl001/ac007/explainer.go

```diff
@@ -5,5 +5,13 @@ import (
 )
 
 func (e *Explainer) explain(ctx context.Context) (string, error) {
-	return "", nil
+	s := `
+Check if the number of host network pods on tenant cluster worker nodes matches the number we expect from k8scloudconfig.
+
+	* Fetch all Tenant Cluster nodes and take the the first worker node by label.
+	* Compare the current pods with host network set with the expected amount of pods on worker node.
+	* See also https://github.com/giantswarm/k8scloudconfig/blob/529491d591e039da1ffde03fef070101c8d4a95c/files/conf/setup-kubelet-environment#L21. 
```

Comment:
This is not the right line number pal. 😛

Reply:
Also, https://github.com/giantswarm/k8scloudconfig/blob/529491d591e039da1ffde03fef070101c8d4a95c/files/conf/setup-kubelet-environment#L26 says `6` but the tests here say `5`. What is it now? That has to got fixed somehow.

Reply:
Forgot to update the explainer and the executor, it's 6 pods as the k8scloudconfig says.

Reply:
Updated the line, thanks 👍